rules/windows/command_and_control_teamviewer_remote_file_copy.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an executable or script file remotely downloaded via a TeamViewer transfer session, verify the context of
+this activity.
```

```suggestion
Identifies an executable or script file remotely downloaded via a TeamViewer transfer session.
```
Unless `verify the context of this activity` is expanded, I don't think it is needed.